ci/openshift-ci/cleanup.sh

```diff
@@ -25,6 +25,10 @@ WORKAROUND_9206_CRIO=${WORKAROUND_9206_CRIO:-no}
 # Ignore errors as we want best-effort-approach here
 trap - ERR
 
+# Delete webhook resources
```

Comment:
Checked that it reverts the resources installed in https://github.com/kata-containers/kata-containers/blob/main/ci/openshift-ci/cluster/deploy_webhook.sh